components/menus/QiskitOrgMenu.vue

```diff
@@ -70,18 +83,15 @@
 
 <script lang="ts">
 import Vue from 'vue'
-import { Component, Prop } from 'vue-property-decorator'
+import { Component } from 'vue-property-decorator'
+
+import { QISKIT_URL, orderedQiskitElements, orderedCommunitySubLinks } from '~/constants/links'
```

Comment:
Can we make the name of the module more explicit (highlighting the fact that it contains the menu items? Example:
```suggestion
import { QISKIT_URL, orderedQiskitElements, orderedCommunitySubLinks } from '~/constants/menuLinks'
```